SharpDenizenTools/MetaObjects/MetaExtension.cs

```diff
@@ -0,0 +1,90 @@
+﻿using FreneticUtilities.FreneticExtensions;
+using SharpDenizenTools.MetaHandlers;
+using System.Collections.Generic;
+
+namespace SharpDenizenTools.MetaObjects
+{
+    /// <summary>A extension to an existing meta object.</summary>
+    public class MetaExtension : MetaObject
+    {
+        /// <inheritdoc/>
+        public override MetaType Type => MetaDocs.META_TYPE_EXTENSION;
+
+        /// <inheritdoc/>
+        public override string Name => ExtensionName;
+
+        /// <summary>The name of the extension.</summary>
+        public string ExtensionName;
+
+        /// <summary>The type and name of the object this extension meta extends.</summary>
+        public string Extend;
```

Comment:
Why is one string holding two separate values? Separate em.

Reply:
Separated into `target_type` and `target_name`, not 100% sure about these names so let me know if you have any suggestions